Automattic/sensei
# Add Welcome email

Resolves #6608 

## Proposed Changes
* Add Welcome email. Sends an email to the student when they start a course.

## Testing Instructions

1. Enable `email_customization` feature flag.
2. Recreate emails in Sensei LMS -> Tools.
3. Create a student, a course.
4. As a student, go to frontend and start the course.
5. Check the student's mailbox.

## Screenshot
<img width="1389" alt="CleanShot 2023-03-10 at 04 42 12@2x" src="https://user-images.githubusercontent.com/329356/224166852-4f610412-6fb1-4f97-ae32-7dc71d364800.png">
<img width="940" alt="CleanShot 2023-03-10 at 04 48 29@2x" src="https://user-images.githubusercontent.com/329356/224167870-f94c6fd4-9e56-43e8-bef0-6bdd566ff989.png">

## Pre-Merge Checklist
